data/interfaces/default/graphs.html

```diff
@@ -258,9 +335,10 @@
             var m = dateValue.getMonth() + 1;
             var y = dateValue.getFullYear();
             var dateString = '' + y + '-' + (m<=9 ? '0' + m : m) + '-' + (d <= 9 ? '0' + d : d);
-
+            var u = '';
+            //var u = $.cookie("example") == 'tabs4') ? 'get_addedat_modal': 'history_table_modal'
             $.ajax({
-                "url": "history_table_modal",
+                "url": "get_addedat_modal",
```

Comment:
The `selectHandler` function is duplicated, here and below L388.

Clicking a series in the "Daily Play count" calls `get_addedat_modal` instead of `history_table_modal`.